### PR TITLE
use stable version 1.0.0 of nikic/fast-route, no code changes since 0.8.0

### PR DESCRIPTION
**Pull request by sandrokeil**
The package maintainer has [nothing changed since 0.8.0](https://github.com/nikic/FastRoute/releases/tag/v1.0.0).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     },
     "require": {
         "php": "^5.5 || ^7.0",
-        "nikic/fast-route": "^0.8.0",
+        "nikic/fast-route": "^1.0.0",
```
Comment (geerteltink) [inline composer.json:22]: Shouldn't it be `^1.0`?

EDIT: nvm, got it mixed up with `~1.0.0`.

**Comment by bakura10** [comment]
Hi,

Can this be merged? ping @weierophinney

File: composer.json (hunk 1 above)
Comment (schnittstabil) [inline composer.json:22]: What about `^0.8.0 || ^1.0.0` ? @sandrokeil already mentioned: nothing has changed.

Comment (Ocramius) [inline composer.json:22]: Nah, stable FTW. Now merging/tagging

**Comment by Ocramius** [comment]
@sandrokeil included this in 1.2.0, thanks!

**Pull request merged**